README.md

```diff
@@ -78,6 +78,49 @@ Further it might be handy to remove e.g. `undefined` tag keys or values. This ca
 {formatting, [{purge, [{tag_keys, undefined}, {tag_values, undefined}]}]}
 ```
 
+### Auto subscriptions:
+
+There is capability for making subscription automatically for each new entry. By default it is off. If you want to use auto
+subscription you should turn it on in reporter options:
```

Comment:
you _need_ to enable it in the reporter options and also provide a callback module which handles newly created entries.